### ISSUE TITLE
Contao 4: Templates not found

### ISSUE THREAD

**Issue by emkayy** [madeyourday/contao-rocksolid-columns#9]
Not so rocksolid in Contao 4:

- Unexpected behavior after saving the first wrapper element: wrapper start, column end, wrapper end, wrapper end elements are created.
- Templates not found:

Warning: include(/www/xyz/contao/vendor/madeyourday/contao-rocksolid-columns/templates/ce_rs_columns_start.html5): failed to open stream: No such file or directory in /www/xyz/contao/vendor/contao/core-bundle/src/Resources/contao/library/Contao/TemplateInheritance.php on line 88

Which isn't surprising, since they are not in /vendor.

**Comment by ausi** [comment]
I was not able to reproduce these issues.

> Which isn't surprising, since they are not in /vendor.

Why aren’t the template files in the *vendor* directory? Did you move them to a different place? How did you install the *rocksolid-columns* extension?

**Comment by emkayy** [comment]
I downloaded the files from the contao-4 branch and copied them like the readme says:
"Install to /system/modules/rocksolid-columns/"

However, even it the template files could be found - after saving a wrapper element, there is still the issue with the multiple "wrong" content elements.

**Comment by ausi** [comment]
The message “Install to */system/modules/rocksolid-columns/*” is meant for Contao 3.X. For Contao 4.X please install the extension `madeyourday/contao-rocksolid-columns` via Composer, this should fix the issues.

You can find more information about our extensions and Contao 4 in our Blog: <http://rocksolidthemes.com/de/contao/blog/contao-4-composer-theme-installieren>

**Comment by emkayy** [comment]
Installation via Composer unfortunately is not possible in this setup (shared hosting).
The Composer plug-in for Contao is still very buggy and installation of extensions is also not working.

**Comment by ausi** [comment]
The installation via Composer is currently required for this extension to work properly. As an alternative you could try to install it into *vendor/madeyourday/contao-rocksolid-columns* and update the autoload files in *vendor/composer* accordingly.

**Comment by discordier** [comment]
I feel I have to jump in here for a moment.

> Installation via Composer unfortunately is not possible in this setup (shared hosting).

Contao itself relies on composer to manage installation of any bundle (to create the symlinks).
How were you able to install Contao 4 without composer? :eyes: 

> The Composer plug-in for Contao is still very buggy and installation of extensions is also not working.

You say the composer-plugin is still very buggy but lack even the slightest hint of what breaks for you.
- What kind of bug are you experiencing in Contao 4?
- Please define "installation of extensions is [also] not working." more specific.
- Please file [issue reports](https://github.com/contao-community-alliance/composer-plugin) (mark it as `[V3]` in the headline pls).

**Comment by emkayy** [comment]
Contao does not rely on Composer anymore since 4.0.2 (https://contao.org/en/news/contao-4_0_2.html)

I tried to install the extension with https://github.com/contao-community-alliance/composer-plugin. For that I fixed the HTTP security issue for legacy extensions. However, I removed the composer plugin again since it produced various errors. Installation of extensions was not possible. Cant' remember the exact error messages.

@discordier I didn't comment any further on the composer errors here since this repository is not related to the composer plugin.

@ausi Thanks for the suggestion, but how do I update the autoload files in vendor/composer?
